observability/logging/cluster-logging-support.adoc

```diff
@@ -47,6 +47,14 @@ LokiStack development is ongoing. Not all APIs are currently supported.
 |RecordingRule
 |recordingrule.loki.grafana/v1
 |Supported in 5.7
+
+|LogFileMetricExporter
+|LogFileMetricExporter.logging.openshift.io/v1alpha1
+|Supported in 5.8
+
+|ClusterLogForwarder
+|clusterlogforwarder.observability.openshift.io/v1
+|Supported in 5.8
```

Comment:
Not sure if this is the best representation but `ClusterLogForwarder` changed API group in 6.0
```suggestion
|ClusterLogForwarder
|clusterlogforwarder.logging.openshift.io/v1
|Supported in 5.8 and 5.9

|ClusterLogForwarder
|clusterlogforwarder.observability.openshift.io/v1
|Supported in 6.0
```

Reply:
I don't have a strong preference, maybe then "Supported from"

Reply:
thanks, updated to "Supported from"